tests/integration/cmd_push_test.go

```diff
@@ -367,7 +367,7 @@ var _ = Describe("odo push command tests", func() {
 				"backend",
 				appName,
 				project,
-				[]string{"stat", filepath.Join(dir, "src", "src", "main", "java", "AnotherMessageProducer.java")},
+				[]string{"stat", filepath.ToSlash(filepath.Join(dir, "src", "src", "main", "java", "AnotherMessageProducer.java"))},
```

Comment:
we need to use flepath.ToSlash() at https://github.com/openshift/odo/pull/3524/files#diff-e977fd6c6dedc4d8ed7367e136764033R378 also